src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -468,12 +468,20 @@ void LottieParser::parsePropertyInternal(T& prop)
 
 
 template<typename T>
-void LottieParser::parseProperty(T& prop)
+void LottieParser::parseProperty(T& prop, LottieSlot::Type type, LottieObject* parent)
```

Comment:
Considering again, the term 'parent' here, does it function conceptually as a parent in a hierarchical structure, or is it simply the 'object' associated with a property? I thin it's more likely to be => LottieObject* obj

Reply:
Yes, it's just associated object, renamed it